packages/lib-content/src/components/Sidebar/Sidebar.js

```diff
@@ -1,4 +1,3 @@
-import Link from 'next/link'
```

Comment:
This component corresponds to DropdownNav which needs the same refactoring. I'll do that before merging.